install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+Labels:				name=busybox-nfs-pod
+Status:				Running
+Reason:				
+Message:			
+IP:				10.1.0.5
+Replication Controllers:	<none>
+Containers:
+  busybox-nfs-pod:
+    Container ID:	docker://346d432e5a4824ebf5a47fceb4247e0568ecc64eadcc160e9bab481aecfb0594
+    Image:		busybox
+    Image ID:		docker://17583c7dd0dae6244203b8029733bdb7d17fccbb2b5d93e2b24cf48b8bfd06e2
+    QoS Tier:
+      cpu:		BestEffort
+      memory:		BestEffort
+    State:		Running
+      Started:		Mon, 21 Mar 2016 10:19:48 -0400
+    Ready:		True
+    Restart Count:	0
+    Environment Variables:
+Conditions:
+  Type		Status
+  Ready 	True 
+Volumes:
+  nfsvol:
+    Type:	NFS (an NFS mount that lasts the lifetime of a pod)
+    Server:	nfs.f22
+    Path:	/opt/nfs
+    ReadOnly:	false
+  default-token-32d2z:
+    Type:	Secret (a secret that should populate this volume)
+    SecretName:	default-token-32d2z
+Events:
+  FirstSeen	LastSeen	Count	From			SubobjectPath				Reason		Message
+  ─────────	────────	─────	────			─────────────				──────		───────
+  4m		4m		1	{scheduler }							Scheduled	Successfully assigned busybox-nfs-pod to ose70.rh7
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Pulled		Container image "openshift3/ose-pod:v3.1.0.4" already present on machine
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Created		Created with docker id 249b7d7519b1
+  4m		4m		1	{kubelet ose70.rh7}	implicitly required container POD	Started		Started with docker id 249b7d7519b1
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Pulled		Container image "busybox" already present on machine
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Created		Created with docker id 346d432e5a48
+  4m		4m		1	{kubelet ose70.rh7}	spec.containers{busybox-nfs-pod}	Started		Started with docker id 346d432e5a48
+----
+[NOTE] 
+====
+Instead of a Persistent Volume Claim being listed under the 'Volumes' section we have 'nfsvol' indicating we used the NFS plugin to access the storage asset.
+====
+
+====
+
+== [Security Requirement] NFS Volume Access
+Access is necessary to a node in the NFS server for either example. On the node examine
+the nfs-fuse mount:
+
+----
+#on a nfs storage node:
+[root@nfs nfs]# ls -lZ /opt/nfs/
+total 8
+-rw-r--r--. 1 root 100003  system_u:object_r:usr_t:s0     10 Oct 12 23:27 test2b
+              <1>
+                     <2>
+----
+<1> the owner has id 0.
+<2> the group has id 100003.
+====
+
+In order to access the NFS mount, the container must match the SELinux label, and
+either run with a UID of 0, or with 100003 in its supplemental groups range. It is recommended to gain
+access to the volume by matching the NFS mount's groups, which will be defined in the pod
+definition below.
+
+By default, SELinux does not allow writing from a pod to a remote NFS server. To enable
+writing to NFS volumes with SELinux enforcing on each node, run:
+
+----
+# setsebool -P virt_sandbox_use_nfs on
```

Comment:
The ../persistent_storage/persistent_storage_nfs.html doc shows setting two se bools:
# setsebool -P virt_use_nfs 1
# setsebool -P virt_sandbox_use_nfs 1